fiftyone/core/collections.py

```diff
@@ -3956,38 +3958,76 @@ def map_samples(
         self,
         map_fcn,
         save=False,
+        skip_failures=False,
+        parallelize_method=None,
```

Comment:
We need to set `parallelize_method=None` here so that `fo.config.default_parallelization_method` will be used